tests/conftest.py

```diff
@@ -53,6 +53,12 @@ def gemini_path(request):
     gemini_db = "tests/fixtures/HapMapFew.db"
     return gemini_db
 
+@pytest.fixture(scope='function')
```

Comment:
Does this decorator tell you when that "big" GEMINI db is not there?

Would it make sense to be a bit smarter here and pull the S3 file when and if it's not present while running the testsuite? Could save some time for new devs/beginners before they figure it out when reading the docs...

Automation >> Docs